main/getting-started/before-using-agoric.md

```diff
@@ -15,42 +16,53 @@ git clone https://github.com/Agoric/agoric-sdk
 cd agoric-sdk
 yarn install
 yarn build
-yarn link-cli ~/bin/agoric
-agoric --version
 ```
 
-Then proceed to [starting a project](/getting-started/start-a-project.md).
+### Add the `agoric` command to your PATH
+
+The `agoric` command is a shell script that uses `yarn` to run the Agoric SDK from the `agoric-sdk` directory. To use it from any directory on MacOS, add it to your path:
+
+```shell
+yarn link-cli ~/bin/agoric # creates an Agoric directory to reference
```

Comment:
That assumes `~/bin` exists. And this step doesn't create any directories. How about:

```suggestion
mkdir -p ~/bin
yarn link-cli ~/bin/agoric # installs a small "agoric" script
```

Reply:
How was this resolved? I'm struggling to find it.